pkgs/development/python-modules/jupyter_server/default.nix

```diff
@@ -77,6 +77,8 @@ buildPythonPackage rec {
   ] ++ lib.optionals stdenv.isDarwin [
     # attempts to use trashcan, build env doesn't allow this
     "test_delete"
+    # server returns wrong status code
```

Comment:
Sounds pretty broken to me :p

Reply:
Yeah, but I assume that it's because of the sandbox.

I checked that jupyterlab starts, but I didn't try anything else.

<img width="1440" alt="Captura de Pantalla 2022-07-14 a la(s) 02 33 42" src="https://user-images.githubusercontent.com/238528/178916507-c268de03-e312-4fb6-b95b-0c371c5ccd01.png">